README.md

```diff
@@ -25,3 +25,46 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+### Runtime Analysis, Maxie M:
+
+The runtime for the algorithm implemented is O(n log n). The recurrence relation for T(n) in this divide and conquer sum will be
+
+$$
+T(n) = \begin{cases} 
+1 & \text{if } n \leq 2 \\
+3T\left(\frac{n}{3}\right) + 1 & \text{if } n > 2 
+\end{cases}
+$$
+
+Now that we have all this information it is time to solve this using substitution.  
+
+The recurrence relation: 
+$T(n) = 3T(\frac{n}{3}) + 1$
+
+$T(n) = 3\left(3T\left(\frac{n}{9}\right) + 1\right) + 1$
+
+$= 9T\left(\frac{n}{9}\right) + 3 + 1$
+
+$= 9\left(3T\left(\frac{n}{27}\right) + 1\right) + (3 + 1)$
+
+$= 27T\left(\frac{n}{27}\right) + 3^2 + (3 + 1)$
+
+$= 27T\left(\frac{n}{27}\right) + 3^2 + 3^1 + 3^0$
+
+$\vdots$
+
+i-th expansion: 
+
+$T(n) = 3^i \cdot T\left(\frac{n}{3^i}\right) + \left(3^{i-1} + 3^{i-2} + \cdots + 3^1 + 1\right)$
+
+$T(n) = 3^i \cdot T\left(\frac{n}{3^i}\right) + \left(3^i - 1\right)$
+
+$T(n) = 3^{\log_3(n)} \cdot T(1) + \sum_{i = 0}^{n} ar = a\left(\frac{1 - r^{n+1}}{1 - r}\right)$
+
+$T(n) = n \cdot T(1) + \frac{(1 - 3^{\log_3(n) + 1})}{1 - 3} = \frac{(1 - 3^{\log_3(n) + 1})}{-2}$
```

Comment:
...so the asymptotic expression is negative?

Reply:
The asymptotic expression is not negative. The reason it appears negative (having a negative denominator) is just a result of simplifying the sum expression, but this doesn't imply a negative runtime! Constants (negative or positive) don't affect the asymptotic growth rate. The overall time complexity is still O(n). 

Also I went to lab last week to have Ali look it over and he was the one who helped me understand that the simplification would have a negative denominator.

Reply:
I don't see how you can end up with negative work.

Reply:
The negative denominator doesn't actually reflect negative work.
As I stated previously the negative denominator is a result of algebraic manipulation during geometric series simplification. It just happens to include a negative factor. This doesn't mean that the actual work (runtime) is negative. The total amount of work remains positive because the asymptotic complexity ( $O(n)$ ) is not concerned with specific constants involved, it's more concerned with the growth of operations.

Reply:
Well you're right, but it's not negative not because we're ignoring that for asymptotic analysis, but because the numerator is also negative, thus making a positive overall. Please also use the correct notation for denoting that the T(n) expression is a member of a particular complexity class.